plugins/services/src/js/columns/QuotaOverviewLimitColumn.tsx

```diff
@@ -97,3 +91,47 @@ export function limitRenderer(group: ServiceGroup) {
     </TextCell>
   );
 }
+
+function itemIsServiceGroup(item: any): item is ServiceGroup {
+  return item.quota && item.quota.limitStatus;
+}
+
+function getLimitAndLimitNumber(
+  servicesCount: number,
+  groupRoleCount: number
+): [string, number] {
+  return [
+    getQuotaLimit({
+      count: servicesCount,
+      groupRoleCount
+    }),
+    servicesCount - groupRoleCount
+  ];
+}
+
+export function limitRenderer(item: ServiceGroup | ServiceTree | Service) {
+  var limit: string = "N/A";
+  var limitNumber: number = 0;
+
+  if (itemIsServiceGroup(item)) {
+    if (!item.quota || !item.quota.serviceRoles) {
+      return <React.Fragment />;
```

Comment:
Why do we have to return a React.Fragment here?

Reply:
If I recall, the caller interface demanded a React element